### ISSUE TITLE
Add documents through file upload or documents/assessment plugin?

### ISSUE THREAD

**Issue by deku-m** [mlebreuil/netbox-contract#48]
Hi,

Is it possible to immediatly attach the document in the contract etc through file upload or linked with documents plugin?
- https://github.com/Kani999/netbox-attachments
- https://github.com/jasonyates/netbox-documents
As i now notice that i could add text or am i doing something wrong?
Where does it get the documents?

![image](https://user-images.githubusercontent.com/120473446/221810808-fc80a5a2-d00a-43b5-ad0a-499694735a7e.png)

**Comment by Kani999** [comment]
Adding files via https://github.com/Kani999/netbox-attachments should be easy. The codebase of netbox-contract should have added a small part of code. 

See https://github.com/Kani999/netbox-attachments/issues/31 
or https://github.com/Kani999/netbox-attachments#enable-attachments-for-custom-plugin-models 

Can I suggest a PR for this?

**Comment by mlebreuil** [comment]
Added the possibility to for plugin to inject content in the contract and invoice forms.  
Tested successfully with netbox-attachments-2.0.1  
Fix will be part of v2.0.8

**Comment by Kani999** [comment]
With #94 will be available only to add plugin content at the right side of the page.
Thus with netbox-attachments only the configuration with `right_page` will be rendered.

- not tested example:
```
PLUGINS_CONFIG = {
    'netbox_attachments': {
        'apps': ['netbox_contract'],
        'display_default': "right_page",
        'display_setting': {'netbox_contract.contract: 'right_page', 'netbox_contract.invoice: 'right_page'}
    }
}
```
